gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/KafkaAvroJobStatusMonitor.java

```diff
@@ -154,6 +154,7 @@ public org.apache.gobblin.configuration.State parseJobStatus(GobblinTrackingEven
         break;
       case TimingEvent.FlowTimings.FLOW_SUCCEEDED:
       case TimingEvent.LauncherTimings.JOB_SUCCEEDED:
+      case TimingEvent.LauncherTimings.JOB_COMPLETE:
```

Comment:
is the alt. to adding this in KafkaAvroJSM to instead have `ExecuteGobblinWorkflowImpl` emit `JOB_SUCCEEDED`?  if that's what gobblin-on-MR emitted, I'd prefer to fix the new producer rather than change the consumer side.